src/unix/solarish/mod.rs

```diff
@@ -1764,6 +1770,10 @@ pub const IP_ADD_MEMBERSHIP: ::c_int = 19;
 pub const IP_DROP_MEMBERSHIP: ::c_int = 20;
 pub const IPV6_JOIN_GROUP: ::c_int = 9;
 pub const IPV6_LEAVE_GROUP: ::c_int = 10;
+pub const IP_ADD_SOURCE_MEMBERSHIP: ::c_int = 70;
+pub const IP_DROP_SOURCE_MEMBERSHIP: ::c_int = 71;
+pub const IP_BLOCK_SOURCE: ::c_int = 72;
+pub const IP_UNBLOCK_SOURCE: ::c_int = 73;
```

Comment:
These are not the correct values for illumos (and presumably Solaris) and cause test failures:
```
bad IP_ADD_SOURCE_MEMBERSHIP value at byte 0: rust: 70 (0x46) != c 23 (0x17)
bad IP_DROP_SOURCE_MEMBERSHIP value at byte 0: rust: 71 (0x47) != c 24 (0x18)
bad IP_BLOCK_SOURCE value at byte 0: rust: 72 (0x48) != c 21 (0x15)
bad IP_UNBLOCK_SOURCE value at byte 0: rust: 73 (0x49) != c 22 (0x16)
```

Reply:
Thanks, should be fixed now

Reply:
Tests pass now.  Thanks